### PR TITLE
Dev/xiangyan/add debugger launcher

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

File: src/LuaDebuggerLauncher/Getting Started.htm [hunk 1]
```diff
@@ -0,0 +1,65 @@
+﻿<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
```
Comment (AArnott) [inline src/LuaDebuggerLauncher/Getting Started.htm]: You should delete this file. It was just in the item template for guidance.

File: src/LuaDebuggerLauncher/RuleProperties.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    /// </summary>
+    [Export]
+    [AppliesTo(ProjectCapabilities.VisualC)]
+    internal partial class RuleProperties : StronglyTypedPropertyAccess
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RuleProperties"/> class.
+        /// </summary>
+        [ImportingConstructor]
+        public RuleProperties([Import(RequiredCreationPolicy = CreationPolicy.Shared)] ConfiguredProject configuredProject)
```
Comment (AArnott) [inline src/LuaDebuggerLauncher/RuleProperties.cs]: The `[Import]` attribute shouldn't be necessary. Especially with the RequiredCreationPolicy specified.Can you remove it to improve readability of the method signature?

File: src/LuaDebuggerLauncher/tools/CommonProjectSystemExtension.targets [hunk 3]
```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (AArnott) [inline src/LuaDebuggerLauncher/tools/CommonProjectSystemExtension.targets]: This file should have come from a Nuget package as well. Did you install  the `microsoft.visualstudio.projectsystem.sdk.tools` package?

File: src/LanguageModel/Classification/Colourizer.cs [hunk 4]
```diff
@@ -257,7 +257,10 @@ private static IEnumerable<TagInfo> GetTokenTagInfoFromLexer(List<Range> ranges,
                     int tokenEnd = token.End < range.End ? token.End : range.End;
                     if (tokenStart < tokenEnd)
                     {
-                        yield return new TagInfo(tokenStart, tokenEnd - tokenStart, SyntaxKindClassifications[token.Kind]);
```
Comment (AArnott) [inline src/LanguageModel/Classification/Colourizer.cs]: Does this change relate to the debugger changes?

Comment (xiangyan99) [inline src/LanguageModel/Classification/Colourizer.cs]: no

File: src/LuaDebuggerLauncher/tools/CommonProjectSystemExtension.targets (hunk 3 above)
Comment (xiangyan99) [inline src/LuaDebuggerLauncher/tools/CommonProjectSystemExtension.targets]: Yes. I did

**Pull request closed without merging**